Source/WebCore/page/Quirks.cpp

```diff
@@ -1618,4 +1618,14 @@ String Quirks::advancedPrivacyProtectionSubstituteDataURLForText(const String& t
     return ""_s;
 }
 
+// DOFUS Touch app (rdar://112679186)
+bool Quirks::needsResettingTransitionCancelsRunningTransitionQuirk() const
+{
+#if PLATFORM(IOS_FAMILY)
+    return needsQuirks() && !linkedOnOrAfterSDKWithBehavior(SDKAlignedBehavior::ResettingTransitionCancelsRunningTransitionQuirk) && IOSApplication::isDOFUSTouch();
```

Comment:
Optional nit: Store this state in a `static` variable? To avoid repeated computations

Reply:
I'm just following patterns in this file :)